client/src/components/Decision.jsx

```diff
@@ -9,8 +11,16 @@ const Decision = ({ handleDecision }) => {
         Now it's time to make a decision about what to do with this emotion. Do
         you want to:{" "}
       </h3>
-      <button onClick={() => makeDecision("Let it go")}>Let it go</button> or{" "}
-      <button onClick={() => makeDecision("Embrace it")}>Embrace it</button>
+      <button className="let-go-btn" onClick={() => makeDecision("Let it go")}>
+        Let it go
+      </button>{" "}
```

Comment:
Extra space?

Reply:
Yes, otherwise the "or" is smashed too closely between the buttons. This felt more intuitive than adjusting the margins for each button. But I'm open to it if that's better practice, lmk.

Reply:
Ahh I see! You could add margin to the right of the button. Space is another way to do it, but it's typically used to apply space between text. 

If the buttons are also next to the each other visually, you can wrap them in a flex container and apply a gap between them. Would be happy to elaborate next time we meet!